engine/enginerpc/engine_impl.cc

```diff
@@ -61,12 +61,14 @@ namespace asio = boost::asio;
 #include "com/centreon/engine/servicegroup.hh"
 #include "com/centreon/engine/statistics.hh"
 #include "com/centreon/engine/statusdata.hh"
+#include "com/centreon/engine/string.hh"
```

Comment:
Can you take a look at why this header is needed.
And if it is not needed a lot, it could be good to use the absl library to make the equivalent.

Reply:
need for string::ctime